CONTRIBUTING

```diff
@@ -0,0 +1,23 @@
+CONTRIBUTIONS and CODE REVIEWS
```

Comment:
This file should be caled `CONTRIBUTING.txt` for consistency.

Reply:
Done.